src/components/Field/Field.tsx

```diff
@@ -358,7 +358,7 @@ export const Field: FunctionComponent<FieldProps> = (props) => {
                         >
                             <Input
                                 {...commonInputProps}
-                                value={localValue !== null ? localValue : (value ? String(value) : '')}
+                                value={localValue !== null ? localValue : (value  && !props.metaError ? String(value) : '')}
```

Comment:
1) It is not clear from the pull request description what the problem was and how it will be fixed, please consider more clear description so the problem is understable from looking at PR without knowing the context
2) It doesn't look right. Check the following gif:
![issue-177](https://user-images.githubusercontent.com/13132449/78554491-eaf72a80-7813-11ea-997f-820deda52b5e.gif)

The `Country` field does not have a validation error, but it's still resetting its initial value.

Reply:
refactored